certified-connectors/Docurain/apiDefinition.swagger.json

```diff
@@ -0,0 +1,248 @@
+            },
+            "headers": {
+              "Content-Type": {
+                "description": "Content-Type",
+                "type": "string"
+              }
+            }
+          }
+        },
+        "summary": "RenderAsync",
+        "operationId": "RenderAsync",
+        "description": "API to render a form asynchronously by specifying the name of a template saved in Docurain beforehand.",
+        "parameters": [
+          {
+            "name": "outType",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Set the output format: pdf, xls(xlsx), png, jpg, jpg, gif, svg",
+            "x-ms-summary": "output format"
+          },
+          {
+            "name": "templateName",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the name of the template you have already saved in Docurain.",
+            "x-ms-summary": "pre-saved template name"
+          },
+          {
+            "name": "X-Docurain-Data-Type",
+            "in": "header",
+            "required": true,
+            "type": "string",
+            "description": "Set the data format : json, csv, tsv",
+            "x-ms-summary": "data format"
+          },
+          {
+            "name": "body",
+            "in": "body",
+            "required": true,
+            "description": "Specify the string data in the format (json/csv/tsv) that you specify for the data format.",
+            "schema": {
+              "type": "string",
+              "description": "Specify the format of the data in the data format.",
+              "title": "data",
+              "format": ""
+            }
+          }
+        ]
+      }
+    },
+    "/apiw/async/{resultId}/status": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
+            "schema": {
+              "type": "object",
+              "properties": {
+                "status": {
+                  "type": "string",
+                  "description": "Processing status of the specified result ID: PROCESSING:Rendering in progress, FAIL:Failure, SUCCESS:Success",
+                  "title": "status"
+                }
+              }
+            }
+          }
+        },
+        "summary": "GetAsyncStatus",
+        "operationId": "GetAsyncStatus",
+        "description": "Get the processing status of the specified result ID.",
+        "parameters": [
+          {
+            "name": "resultId",
+            "in": "path",
+            "required": true,
+            "type": "string",
+            "description": "Specify the result ID returned by RenderAsync.",
+            "x-ms-summary": "Result ID to query processing status"
+          }
+        ]
+      }
+    },
+    "/apiw/async/{resultId}": {
+      "get": {
+        "responses": {
+          "default": {
+            "description": "default",
```

Comment:
@geckoR42 Please replace the "default" responses with expected status codes, otherwise users may see errors about incorrect schema instead of real error responses from your service in OpenAPI mode.

Reply:
@sriyen-msft Removed "default". Fixed.